mlrun/model_monitoring/db/tsdb/tdengine/tdengine_connector.py

```diff
@@ -112,11 +112,11 @@ def write_application_event(
         """
 
         table_name = (
-            f"{self.project}_"
+            # f"{self.project}_"
             f"{event[mm_schemas.WriterEvent.ENDPOINT_ID]}_"
             f"{event[mm_schemas.WriterEvent.APPLICATION_NAME]}_"
```

Comment:
The underscore looks extraneous:
```suggestion
            f"{event[mm_schemas.WriterEvent.APPLICATION_NAME]}"
```